### PR TITLE
Update the network_tester example

### PR DESCRIPTION
**Pull request by jenselofsson**
Update the network tester with some more features:
Multiple input tensors
Multiple output tensors
Output is printed in json format
Can call invoke() more than once using the NUM_INFERENCES

Updated README

### CODE REVIEW

**Comment by jenselofsson** [comment]
Ping for review

**Comment by jenselofsson** [comment]
Ping for review!

**Comment by jenselofsson** [comment]
@petewarden @wangtz Ping for review

**Comment by gbaned** [comment]
@jenselofsson Can you please resolve conflicts? Thanks!

**Comment by jenselofsson** [comment]
@gbaned Conflicts resolved!

File: tensorflow/lite/micro/examples/network_tester/network_model.h [hunk 1]
```diff
@@ -64,4 +67,4 @@ const unsigned char network_model[] = {
     0x08, 0x00, 0x07, 0x00, 0x06, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x11};
 const unsigned int network_model_len = 576;
 
-#endif  // TENSORFLOW_LITE_MICRO_EXAMPLES_NETWORK_TESTER_NETWORK_MODEL_H_
```
Comment (wangtz) [inline tensorflow/lite/micro/examples/network_tester/network_model.h:67]: Plz keep this.

**Review by wangtz — DISMISSED** [review]

**Review by wangtz — APPROVED** [review]

**Pull request merged**